src/main/java/algorithms/math/InterestCalculator.java

```diff
@@ -0,0 +1,67 @@
+package algorithms.math;
+
+import java.util.Scanner;
+
+public class IntrestCalculator {
+  public static void main(String args[]) throws Exception {
+    int a;
+    double b, c;
+    Scanner sc = new Scanner(System.in);
```

Comment:
add empty new line here similar after variable declaration, change a,b,c to meaningful name